miniapp/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <domain-config cleartextTrafficPermitted="true">
+	<domain includeSubdomains="true">10.0.2.2</domain> <!--localhost-->
```

Comment:
I didn't want to allow every url so if someone uses other http address than localhost, they'd need to put the address in this file.

Reply:
This one is gud but the spec declares that http is allowance. @corycaywood What do you think?

Reply:
I think we shouldn't set the `network-security-config` on the SDK side. We should require the Host App to control this.

So probably it's better to move this file to `testapp`, and then we can set it to allow clear text for all domains. The demo app is only used for testing, so I think this should be okay.

@minh-rakuten @imaginaris Do you think that makes sense?

Reply:
If we want to make it available for every domain then I think using `android:usesCleartextTraffic="true"` in `testapp` is enough.

Reply:
Got it!